package.json

```diff
@@ -53,6 +53,7 @@
     "bluebird": "3.x",
     "continuation-local-storage": "3.x",
     "dateformat": "^3.0.3",
+    "find-root": "^1.1.0",
```

Comment:
does this need to be a real dep rather than a dev dep? isn't debugging for dev?

Reply:
Yeah good catch.

Down the road, we may need to make something like this a real dependency if we ever want to be able to run Appium on non-compiled code.